installation/includes/02_helpers.sh

```diff
@@ -28,31 +41,231 @@ _download_file_from_google_drive() {
   echo "Downloaded from Google Drive ID ${GD_SHARING_ID} into ${TAR_FILENAME}"
 }
 
-get_onboard_audio() {
-  if grep -q -E "^dtparam=([^,]*,)*audio=(on|true|yes|1).*" ${RPI_BOOT_CONFIG_FILE}
```

Comment:
Did you move that check?

Reply:
Yes, i moved and inlined it in https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/1b53fbe9a017f9decbe643c9c5d3f65151255d92/installation/routines/set_raspi_config.sh#L21

The call to the function was only made in this place. 
Also its rahter special and a simliar string is used after the check again.